### PR TITLE
minor improvements to the SDXL doc.

### PR DESCRIPTION
**Pull request by sayakpaul**

### CODE REVIEW

File: docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx [hunk 1]
```diff
@@ -82,6 +82,28 @@ image = pipe(prompt=prompt, output_type="latent" if use_refiner else "pil").imag
...
+init_image = load_image(url).convert("RGB")
+prompt = "a photo of an astronaut riding a horse on mars"
+image = pipe(prompt, image=init_image).images[0]
+```
+
+| Original Image | Refined Image |
+|---|---|
+| ![](https://huggingface.co/datasets/diffusers/docs-images/resolve/main/sd_xl/init_image.png) | ![](https://huggingface.co/datasets/diffusers/docs-images/resolve/main/sd_xl/refined_image.png) |
+
 ### Loading single file checkpoitns / original file format
```
Comment (bghira) [inline docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx:107]: checkpoints - typo

File: docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx [hunk 2]
```diff
@@ -82,6 +82,28 @@ image = pipe(prompt=prompt, output_type="latent" if use_refiner else "pil").imag
...
+
+pipe = StableDiffusionXLImg2ImgPipeline.from_pretrained(
+    "stabilityai/stable-diffusion-xl-refiner-0.9", torch_dtype=torch.float16
+)
+pipe = pipe.to("cuda")
+url = "https://huggingface.co/datasets/patrickvonplaten/images/resolve/main/aa_xl/000000009.png"
+
+init_image = load_image(url).convert("RGB")
+prompt = "a photo of an astronaut riding a horse on mars"
+image = pipe(prompt, image=init_image).images[0]
```
Comment (bghira) [inline docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx:100]: an improvement would be to add the defaults for guidance_scale, width, height, and num_inference_steps so that they are more accessible to new persons.

File: docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx [hunk 3]
```diff
@@ -21,7 +21,7 @@ The abstract of the paper is the following:
 ## Tips
 
 - Stable Diffusion XL works especially well with images between 768 and 1024.
-- Stable Diffusion XL output image can be improved by making use of a refiner as shown below
+- Stable Diffusion XL output image can be improved by making use of a refiner as shown below.
 
 ### Available checkpoints:
```
Comment (bghira) [inline docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx:26]: are these changes intentional? i'm not familiar with the file format.

File: docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx (hunk 2 above)
Comment (sayakpaul) [reply docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx:101]: Thanks, but we like to keep these examples as minimal as possible.

File: docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx (hunk 3 above)
Comment (sayakpaul) [reply docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx:26]: Which fileformat?

Comment (bghira) [reply docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx:26]: mdx files

Comment (sayakpaul) [reply docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.mdx:26]: We use mdx files for documentation.
